core/vtk/ttkDiscreteGradient/ttkDiscreteGradient.cpp

```diff
@@ -242,6 +242,18 @@ int ttkDiscreteGradient::RequestData(vtkInformation *ttkNotUsed(request),
   auto outputCriticalPoints = vtkPolyData::GetData(outputVector, 0);
   auto outputGradientGlyphs = vtkPolyData::GetData(outputVector, 1);
 
+  auto triangulation = ttkAlgorithm::GetTriangulation(input);
+  if(!triangulation) {
```

Comment:
could you wrap these tests into one test function in `core/base/common/MPIUtils.h`?
that way, for the calling module, that would be only a one-liner to call each time.

Reply:
I wrote this in the method `continueComputation` in ttkAlgorithm, as `MPIUtils.h` is for MPI related code.

Reply:
that's great @eve-le-guillou. that does the trick exactly.
one thing though: could you change the name of the function such that it becomes more obvious for other developers that this function is MPI-related (I'm thinking of those who will want to port their module to MPI).
maybe something like:
`checkMPIinput`